Makefile

```diff
@@ -21,7 +27,7 @@ up:
 
 up-with-data:
 	docker-compose -f docker-compose.yml -f docker-compose.seed.yml build
-	docker-compose -f docker-compose.yml -f docker-compose.seed.yml up
+	docker-compose -f docker-compose.yml -f docker-compose.seed.yml up -d
```

Comment:
Good that this is consistent with `make up`, but I feel like this is not-consistent with our other applications. I feel like `make up` should block.

Reply:
Moved both up-with-data and up to be blocking.